### PR TITLE
i144 add max widths for desktop

### PR DESCRIPTION
**Pull request by aidan-0**
## Change Summary
Added max-width for modals and added slight margin for events page on desktop only

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
Is that margin sizing correct?

### CODE REVIEW

**Comment by aidan-0** [comment]
Looks like team tests are failing. I had to update the NewTeamModal to match what it said main had.

File: client/src/views/EventsView.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="!isLoading">
+  <div class="container" v-if="!isLoading">
```
Comment (ChantelleKerr) [inline client/src/views/EventsView.vue:2]: I think just changing it from a div to a v-container should work
` <v-container v-if="!isLoading"> `

File: client/src/views/EventsView.vue [hunk 2]
```diff
@@ -111,4 +111,11 @@ function openEditModal(id: number) {
   padding-bottom: 0px;
   margin-top: -19px;
 }
+
+@media (min-width: 960px) {
+  .container {
+    margin-left: 15px;
+    margin-right: 15px;
+  }
+}
```
Comment (ChantelleKerr) [inline client/src/views/EventsView.vue:120]: I dont think we would need this if we change to use a v-container. Let me know if it works or not

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
I think the events page is still to wide on desktop, I added a few suggestions and hope it works.

**Review by ChantelleKerr — APPROVED** [review]
Looks good :) Approved

**Pull request merged**